src/views/show-control/TakeoffAreaSetupDialog.jsx

```diff
@@ -1,15 +1,15 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { connect } from 'react-redux';
-import { Translation, withTranslation } from 'react-i18next';
-
 import Box from '@material-ui/core/Box';
 import Button from '@material-ui/core/Button';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Switch from '@material-ui/core/Switch';
+import { SelectAll } from '@material-ui/icons';
```

Comment:
We usually import icons directly from the corresponding file (i.e. `@material-ui/icons/SelectAll` in this case). To be honest I don't know whether it's necessary - the reason why we do it this way is to avoid bundling icons from `@material-ui/icons` that we don't need. It used to help in the past when tree-shaking routines were not as smart as they are today.

Reply:
Ah, auto-import. Sorry, I didn't notice it! Fixed.